pkg/providers/gcpsecrets/gcpsecrets.go

```diff
@@ -33,9 +31,10 @@ func New(cfg api.StaticConfig) *provider {
 
 	version := cfg.String("version")
 	if version == "" {
-		p.version = "latest"
-	} else {
-		p.version = version
+		version = "latest"
+	}
+	return &provider{
+		version: version,
```

Comment:
@mumoshu return? how about the below code?

Reply:
This seems terribly wrong! Let's revert this part so that the rest of the New func works !

Reply:
It looks like several different changes got merged in this file on the same day. The original PR looks clean to me
https://github.com/abursavich/vals/commit/67f05de6f50052f02eb280271783013a4ce6a2e4. Sorry for the trouble.

Reply:
Here's a fix: https://github.com/helmfile/vals/pull/128